1.7/administration/installing/cloud/aws/advanced-templates.md

```diff
@@ -0,0 +1,245 @@
+
+    ![Monitor stack creation](../img/dcos-aws-step3a.png)
+
+    ![DC/OS dashboard](../img/ui-dashboard.gif)
+
+1.  Click the dropup menu on the lower-left side to install the DC/OS [Command-Line Interface (CLI)][2]. You must install the CLI to administer your DCOS cluster.
+
+    ![install CLI](../img/ui-dashboard-install-cli.gif)
+    
+## Next steps
+
+Now that your advanced template DC/OS installation is up and running you can
+
+### Add more agent nodes
+
+You can add more agent nodes by creating a new stack by using the [advanced-priv-agent.json]() or [advanced-pub-agent.json]() templates. These templates create agents which are then attached to the `PrivateAgentStack` or `PublicAgentStack` as a part of an AutoScalingGroup. 
+
+Use the output values from the `zen.sh` script and your Master and Infra stacks. These new agent nodes will automatically be added to your DC/OS cluster. 
+
+Private agents:
+
+*  **InternalMasterLoadBalancerDnsName** Specify the `InternalMasterLoadBalancerDnsName` value from your master stack (`<stack-name>-MasterStack-<stack-id>`). You can find this value in the **Outputs** tab.
+*  **KeyName** Specify your AWS EC2 Key Pair. 
+*  **PrivateAgentInstanceCount** Specify the number of private agents.
+*  **PrivateAgentInstanceType** Specify the Amazon EC2 instance type for the private agent nodes. The <a href="https://aws.amazon.com/ec2/pricing/" target="_blank">m3.xlarge</a> instance type is recommended.
+*  **PrivateAgentSecurityGroup** Specify the security group ID for private agents. This group should have limited external access. You can find this value in the **Outputs** tab of the Infrastructure stack (`<stack-name>-Infrastructure-<stack-id>`).
+*  **PrivateSubnet** Specify the `Private SubnetId` output value from the `zen.sh` script. This subnet ID will be used by all private agents. 
+
+Public agents:
+
+*  **InternalMasterLoadBalancerDnsName** Specify the `InternalMasterLoadBalancerDnsName` value from your master stack (`<stack-name>-MasterStack-<stack-id>`). You can find this value in the **Outputs** tab.
+*  **KeyName** Specify your AWS EC2 Key Pair. 
+*  **PublicAgentInstanceCount** Specify the number of public agents.
+*  **PublicAgentInstanceType** Specify the Amazon EC2 instance type for the public agent nodes. The <a href="https://aws.amazon.com/ec2/pricing/" target="_blank">m3.xlarge</a> instance type is recommended.
+*  **PublicAgentSecurityGroup** Specify the security group ID for public agents. This group should have limited external access. You can find this value in the **Outputs** tab of the Infrastructure stack (`<stack-name>-Infrastructure-<stack-id>`).
+*  **PublicSubnet** Specify the `Public SubnetId` output value from the `zen.sh` script. This subnet ID will be used by all public agents. 
+
+## Add more masters
```

Comment:
Are you sure we can add more masters to an existing DCOS cluster ? Can you please confirm this with the DCOS team. I think the configurability in advanced templates only lets you pick the number of master at cluster creation time, but not add more masters once you already have a cluster.